### DISCUSSION TITLE
Selenium automation - strange behaviour

### DISCUSSION THREAD

**Discussion by jsa34** [react-dropzone/react-dropzone#1199] [Q&A]
Hello!

I wondered if you might be able to help with some weird behaviour I am seeing when writing automated tests for an application using this dependency.

In summary:

1. Submit a new file upload (via selenium's native file upload)
2. Click the delete button
3. Upload the same file again

No matter the wait/sleep between interactions, the same behaviour happens: The first upload is successful, the delete clears the file list on the front end, and the subsequent upload results in two of the same file appearing on the user's list

This does not occur manually.

There are other weird behaviours when using selenium's file upload (e.g. not respecting the same file limits as set and tested manually for max uploads; allowing duplicate files with the same filename, which doesn't happen manually)

**Comment by rolandjitsu** [comment]
I have a feeling that it may be related to the internal state of the dropzone. I'd need to refresh my memory on the impl. to be able to tell exactly what's going on. You could try to open again and select nothing (just close), that may clear the state.
